ccm_web/package.json

```diff
@@ -30,6 +30,7 @@
     "@nestjs/passport": "^8.0.1",
     "@nestjs/platform-express": "^8.2.0",
     "@nestjs/sequelize": "^8.0.0",
+    "@nestjs/serve-static": "^2.2.2",
```

Comment:
Looks like this library defaults settings for caching is enabled to true. https://github.com/nestjs/serve-static/blob/master/lib/interfaces/serve-static-options.interface.ts